weaviate/util.py

```diff
@@ -244,19 +244,19 @@ def is_object_url(url: str) -> bool:
 
 def get_valid_uuid(uuid: Union[str, uuid_lib.UUID]) -> str:
     """
-    Validate and extract the UUID.
+    Validate and extract the UUID, excluding any dashes within.
 
     Parameters
     ----------
     uuid : str or uuid.UUID
         The UUID to be validated and extracted.
         Should be in the form of an UUID or in form of an URL (weaviate 'beacon' or 'href').
         E.g.
-        'http://localhost:8080/v1/objects/fc7eb129-f138-457f-b727-1b29db191a67'
+        'http://localhost:8080/v1/objects/fc7eb129f138457fb7271b29db191a67'
         or
-        'weaviate://localhost/28f3f61b-b524-45e0-9bbe-2c1550bf73d2'
+        'weaviate://localhost/28f3f61bb52445e09bbe2c1550bf73d2'
         or
-        'fc7eb129-f138-457f-b727-1b29db191a67'
+        'fc7eb129f138457fb7271b29db191a67'
 
```

Comment:
Here you can actually leave both examples, with dashes and without. This will tell our users that both versions are fine.